pallets/frequency-tx-payment/src/capacity_stable_weights.rs

```diff
@@ -10,28 +10,49 @@
 	unused_imports
 )]
 
-use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use frame_support::{traits::Get, weights::{Weight, constants::{RocksDbWeight, WEIGHT_REF_TIME_PER_NANOS}}};
 use sp_std::marker::PhantomData;
 
+/// The base fee for extrinsics is calculated by running benchmarks.
+/// Capacity needs the base fee to remain stable and not change when benchmarks are run.
+/// CAPACITY_EXTRINSIC_BASE_WEIGHT is a snapshot of the base fee at the time of writing,
```

Comment:
Would be nice to specify the exact date this weight was computed

Reply:
I've added the date and the benchmark commands to the comments.